website/content/platform/faqs/platform_vs_sdk.md

```diff
@@ -0,0 +1,136 @@
+
+The OpenBB Terminal is not installed with the Platform. The Terminal will be reimagined with the new extension framework, it won't get left behind for long!
+
+### Extension Framework
+
+The extension framework lets users install or uninstall individual data and toolkit extensions. No more bloated environments.
+
+```bash
+pip install openbb-yfinance
+
+pip uninstall openbb-yfinance
+```
+
+### Import Statement
+
+Initialize the OpenBB Platform with a shorter statement and reduced import time.
+
+```python
+from openbb import obb
+```
+
+### REST API Compliant
+
+The OpenBB Platform is built for REST API deployments, out-of-the-box. Outputs are JSON serializable, and this is a major difference between the SDK and Platform.
+
+Start the API on `localhost` with:
+
+```bash
+uvicorn openbb_core.api.rest_api:app
+```
+
+### Verbose Namespaces
+
+After careful consideration, the decision was made to name functions with more verbosity. This adds clarity to the functions and lets the user better understand its purpose. It also improves the performance of AI tooling built on top of the Platform.
+
+```python
+obb.equity.fundamental.employee_count("AAPL")
+```
+
+### Asset Class Names
+
+Some asset classes have been renamed:
+
+- stocks -> equity
+- forex -> currency
+- stocks.options -> derivatives.options
+- futures -> derivatives.futures
+
+### Source -> Provider
+
+V3 SDK users should note that the `source` parameter is now, `provider`.
+
+:::note
+OpenBB Hub Preferences for default sources do not currently sync with V4 Platform, and some API keys not used in the Terminal application are not able to be saved to the Hub.
+:::
+
+### API Key Management
+
+API keys and user preferences are stored in a JSON file - `$HOME/.openbb_platform` - instead of the `ENV` file in `$HOME/.openbb_sdk`.
+
+Credentials can be entered directly from the Python interface:
+
+```python
+from openbb import obb
+obb.user.credentials.fmp_api_key="REPLACE_WITH_YOUR_KEY"
+```
+
+### OpenBB Hub Login
+
+Login to your OpenBB Hub account with an email/password combo or a revokable Personal Access Token (recommended):
+
+```python
+from openbb import obb
+obb.account.login()
+```
+
+### Function Outputs
+
+The default output format can be selected by the user, and all outputs are Pydantic models. If you are transitioning from V3 SDK and like working with Pandas DataFrames, set the preference to "dataframe" to get a V3-like response.
+
+```python
+from openbb import obb
+obb.user.preferences.output_type="dataframe"
+df = obb.equity.price.historical("AAPL", provider="yfinance")
+```
+
+When the `output_type` is set to, `OBBject`, DataFrames are created from the response object after it is returned.
+
+```python
+from openbb import obb
+data = obb.equity.price.historical("AAPL", provider="yfinance")
+df = data.to_df()
+```
+
+### Jupyter
+
+Jupyter does not get installed with the OpenBB Platform. Install from `pip` in the environment containing the Platform.
```

Comment:
Although it can be if you install `openbb-devtools`!

Reply:
Would need to make Jupyter a dependency there.  Currently it just has `ipykernel`

```
[tool.poetry.dependencies]
python = ">=3.8,<3.12"               # scipy forces <4.0 explicitly
ruff = "^0.1.5"
pylint = "^3.0.2"
mypy = "^1.6.1"
pydocstyle = "^6.3.0"
black = "^23.11.0"
bandit = "^1.7.5"
pre-commit = "^3.5.0"
tox = "^4.11.3"
pytest = "^7.4.3"
pytest-cov = "^4.1.0"
ipykernel = "^6.26.0"
types-python-dateutil = "^2.8.19.14"
types-toml = "^0.10.8.7"
poetry = "^1.7.0"
```

Reply:
gotcha! we can have it! 
what do you think, is it worth having?
@piiq thoughts?

Reply:
I don't think it's worth having jupyter as a dependency it there.

Additionally I would be cautious with pointing ppl to the jupyter metapackage. It hasn't been updated for a few years and come classifiers are clearly outdated

<img width="490" alt="image" src="https://github.com/OpenBB-finance/OpenBBTerminal/assets/11668535/3c926419-52f9-4abc-97c5-6d75b2d32dc3">


Jupyter frontends are very different from each other and there are different packages that ship them. 
For example a `pip install jupyter` command would get you ipython, jupyter notebook (a.k.a. classic notebook) and qtconsole (which would pull in pyside which is a pretty large package).
Alternatively you can install `jupyter-notebook` which will just get you ipython and the classic notebook, then you can install jupyter-lab which would get you the jupyter lab we all are used to, or you can install retrolab which is a jupyter-notebook like ui but built on top of the tooling that's developed for jupyter lab.

I don't know what's the best solution for this part of the docs. I personally would probably tell ppl to install jupyter-lab or retrolab or use jupyter kernels from vscode with the ipykernel package